recipes/jarvis-tools/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "jarvis-tools" %}
+{% set version = "2020.10.10" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jarvis-tools-{{ version }}.tar.gz
+  sha256: b4cc45267100fe2cc4d114d1453ad39e87d35244ce511c44b1fc99ca277f8a07
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - joblib
+    - matplotlib-base
+    - numpy
+    - pytest
+    - python
+    - requests
+    - scipy
+    - spglib
+    - toolz
+    - xmltodict
+
+test:
+  imports:
+    - jarvis
+    - jarvis.ai
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://jarvis.nist.gov
+  summary: 'jarvis-tools: an open-source software package for data-driven atomistic materials design'
+  license: NIST-PD
```

Comment:
I am a bit confused about the license here. The LICENSE file you included is not identical to https://spdx.org/licenses/NIST-PD.html (the one you included essentially has that extra paragraph at the end.)

Reply:
Sorry, the official [Jarvis license file](https://github.com/usnistgov/jarvis/blob/master/LICENSE) is mangled. Thanks for checking it.

However, the official NIST license is [here](https://www.nist.gov/open/copyright-fair-use-and-licensing-statements-srd-data-software-and-technical-series-publications). It's close, but not exactly what's listed at [spdx.org](https://spdx.org/licenses/NIST-PD.html). I think the official location for the license is at NIST itself though.

Reply:
I've update to the official NIST license and add an issue to fix upstream, https://github.com/usnistgov/jarvis/issues/127

Reply:
Actually the license listed at spdx.org is totally mangled. It's wrong and needs to be fixed.